java/src/Example.java

```diff
@@ -0,0 +1,33 @@
+import humbug.HumbugConsent;
```

Comment:
Better to put `Example.java` in a separate package so that it doesn't get released on Maven.